src/stream.cpp

```diff
@@ -1886,6 +1886,10 @@ namespace stream {
 #if defined SUNSHINE_TRAY && SUNSHINE_TRAY >= 1
           system_tray::update_tray_pausing(proc::proc.get_last_run_app_name());
 #endif
+          if(config::video.dd.config_revert_on_disconnect)
```

Comment:
```
    if (--running_sessions == 0) {
        bool revert_display_config{config::video.dd.config_revert_on_disconnect};
        if (proc::proc.running()) {
#if defined SUNSHINE_TRAY && SUNSHINE_TRAY >= 1
          system_tray::update_tray_pausing(proc::proc.get_last_run_app_name());
#endif
        } else {
            // We have no app running and also no clients anymore.
            revert_display_config = true;
        }

        if (revert_display_config) {
            display_device::revert_configuration();
        }

        platf::streaming_will_stop();
      }
```

Could we have it like this? It's my preference only and I'm also fine with the original impl.